packages/react/src/QuestionnaireForm/QuestionnaireForm.tsx

```diff
@@ -529,3 +633,11 @@ export function isQuestionEnabled(
     return true;
   }
 }
+
+function getNumberOfPages(items: QuestionnaireItem[]): number {
+  const pages = items.filter((item) => {
+    const extension = getExtension(item, 'http://hl7.org/fhir/R4B/extension-questionnaire-itemcontrol.html');
```

Comment:
URL for this extension should have been
http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl


[reference](http://hl7.org/fhir/R4/extension-questionnaire-itemcontrol.html)